docs/AI-for-security/attack-discovery.asciidoc

```diff
@@ -37,15 +37,36 @@ This page describes:
 [[attack-discovery-rbac]]
 == Role-based access control (RBAC) for Attack Discovery
 
-The `Attack Discovery: All` privilege allows you to use Attack Discovery.
+You need the `Attack Discovery: All` privilege to use Attack Discovery.
 
 image::images/attck-disc-rbac.png[Attack Discovery's RBAC settings,60%]
 
+[discrete]
+== Set up Attack Discovery
+
+By default, Attack Discovery analyzes up to 100 alerts from the last 24 hours, but you can customize how many and which alerts it analyzes using the settings menu. To open it, click the gear icon next to the **Generate** button.
+
+image::images/security-attack-discovery-settings.png[Attack Discovery's settings menu,60%]
+
+You can select which alerts Attack Discovery will process by filtering based on a KQL query, the time and date selector, and the **Number of alerts** slider. Note that sending more alerts than your chosen LLM can handle may result in an error. Under **Alert summary** you can view a summary of the selected alerts grouped by various fields, and under **Alerts preview** you can see more details about the selected alerts.
+
+[NOTE]
+====
+How to add non-ECS fields to Attack Discovery
```

Comment:
```suggestion
*How to add non-ECS fields to Attack Discovery*

```

Reply:
Another option is to use the sidebar component, like you did [here](https://github.com/elastic/security-docs/pull/6693/files#diff-33d535d01147ac696a266737a748eb8606ff5c95590a089e6689139f192dc6b1R27-R36).